### DISCUSSION TITLE
I have a problem with inputs that are of Array type

### DISCUSSION THREAD

**Discussion by shon2789** [graphql/graphiql#3551] [Q&A / Support]
Hi, i'm using GraphiQL to render my Schema explorer, I have a problem with inputs that are Arrays, the explorer instead of opening [] it opens {}, it messes up our logic of parsing the filters of the query, is there any solution for it or any guide I can use to implement it?

<img width="1491" alt="Screenshot 2024-03-06 at 10 29 23" src="https://github.com/graphql/graphiql/assets/74869982/d9855b92-1c3a-4f1e-9523-3e285ce6dc37">

<img width="1658" alt="Screenshot 2024-03-06 at 10 29 10" src="https://github.com/graphql/graphiql/assets/74869982/2213a261-babb-45bd-ab3a-5e0ac4cc7dfe">

**Comment by acao** [comment]
we are considering "adopting" the explorer, which may be required to fix this bug
